.github/workflows/python.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Install
         run: |
           pip install -U pip
-          pip install .[numpy,tensorflow,paddlepaddle]
```

Comment:
Why is this being removed?

Reply:
Version clash with tensorflow on protobuf. Taking the most recent library there.